core/ui/EditTemplate/title.tid

```diff
@@ -35,6 +35,10 @@ tags: $:/tags/EditTemplate
 
 <$checkbox tiddler="$:/config/RelinkOnRename" field="text" checked="yes" unchecked="no" default="no"> {{$:/language/EditTemplate/Title/Relink/Prompt}}</$checkbox>
 
+<$set name="backlinks" filter="[title<fromTitle>backlinks[]]">
+
+<$reveal type="nomatch" default=<<backlinks>> text="">
+
```

Comment:
Please also check the DOM. There are several paragraph elements, that shouldn't be there. Some text nodes that contain no text and so on.

Reply:
hey @pmario, Im not exactly familiar with that - I just tried to follow the convention of the file - I guess that resulted in some extra empty elements. Its never really clear to me when a new line will show up or not. Any help you can provide would be greatly appreciated!

Reply:
Hey Mario - just a bump on this. I'm eager to try and contribute something to TW!